recipes/annembed/build.sh

```diff
@@ -1,8 +1,42 @@
 #!/bin/bash -euo
 
-# Add workaround for SSH-based Git connections from Rust/cargo.  See https://github.com/rust-lang/cargo/issues/2078 for details.
-# We set CARGO_HOME because we don't pass on HOME to conda-build, thus rendering the default "${HOME}/.cargo" defunct.
-export CARGO_NET_GIT_FETCH_WITH_CLI=true CARGO_HOME="$(pwd)/.cargo"
+# Workaround for SSH-based Git connections from Rust/cargo
+# We set CARGO_HOME because conda-build doesn't pass HOME
+export CARGO_NET_GIT_FETCH_WITH_CLI=true
```

Comment:
This is not needed since Rust 1.71, I believe. You can delete this line and its comment above